plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/taxonomy-controls/index.tsx

```diff
@@ -48,6 +48,20 @@ function TaxonomyControls( {
 		return null;
 	}
 
+	/**
+	 * Normalize the name so first letter of every word is capitalized.
+	 */
+	const normalizeName = ( name: string ) => {
+		if ( ! name ) {
+			return name;
+		}
```

Comment:
If we expect that name could be undefined or null, I would update the type of the name parameter to:
```
name: string | undefined | null
```

Also, I would update the return to:

```
if ( ! name ) {
    return "";
}
```

Reply:
Good call out!